packages/storybook/package.json

```diff
@@ -18,8 +18,9 @@
     "@storybook/core": "^6.3.12",
     "@storybook/storybook-deployer": "^2.8.10",
     "@storybook/web-components": "^6.3.12",
-    "@vaadin/vaadin-button": "^3.0.0-alpha1",
-    "@vaadin/vaadin-notification": "^2.0.0-alpha1",
+    "@vaadin/button": "^23.1.1",
+    "@vaadin/notification": "^23.1.1",
+    "@vaadin/progress-bar": "^23.1.1",
```

Comment:
Unnecessary dependency added?

Reply:
You're right. `@vaadin/progress-bar` isn't loaded directly from the storybook package. I'll remove it.